content/docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Getting Started with Prometheus
+sort_rank: 1
+---
+
+# What is prometheus ?
+
+Prometheus is a system monitoring and alerting system. It was opensourced by SoundCloud in 2012 and was incubated by Cloud Native Computing Foundation. Prometheus stores all the metrics data as time series, i.e metrics information is stored along with the timestamp at which it was recorded, optional key-value pairs called as labels can also be stored along with metrics.
+
+# What are metrics and why is it important?
+
+Metrics in layperson terms is a standard for measurement. What we want to measure depends from application to application. For a web server it can be request times, for a database it can be CPU usage or number of active connections etc.
+
+Metrics play an important role in understanding why your application is working in a certain way. If you run a web application and someone comes up to you and says that the application is slow. You will need some information to find out what is happening with your application. For example the application can become slow when the number of requests are high. If you have the request count metric you can spot the reason and increase the number of servers to handle the heavy load. Whenever you are defining the metrics for your application you must put on your detective hat and ask this question **what all information will be important for me to debug if any issue occurs in my application?**
+
+# Basic Architecture of Prometheus
+
+The basic components of prometheus are:
+
+- Prometheus Server (The server which scrapes and stores the metrics data).
+- Instrumented Application which expose its metrics / Exporter which exposes metrics of another application.
```

Comment:
```suggestion
- Instrumented Application which exposes its metrics, or
- Exporter which exposes metrics of another application.
```

Reply:
should this be separate?

Reply:
as per call: Can also say "targets to be scraped, for example:"